diff --git a/vendor-cookbooks/phpmyadmin/recipes/default.rb b/vendor-cookbooks/phpmyadmin/recipes/default.rb
index 1d97a38..c264a11 100644
--- a/vendor-cookbooks/phpmyadmin/recipes/default.rb
+++ b/vendor-cookbooks/phpmyadmin/recipes/default.rb
@@ -22,11 +22,11 @@ require 'digest/sha1'
 
 # PHP Recipe includes we already know PHPMyAdmin needs
 if node['phpmyadmin']['stand_alone'] then
-	include_recipe 'php'
-	include_recipe 'php::module_mbstring'
-	include_recipe 'php::module_mcrypt'
-	include_recipe 'php::module_gd'
-	include_recipe 'php::module_mysql'
+	# include_recipe 'php'
+	# include_recipe 'php::module_mbstring'
+	# include_recipe 'php::module_mcrypt'
+	# include_recipe 'php::module_gd'
+	# include_recipe 'php::module_mysql'
 
 	directory node['phpmyadmin']['upload_dir'] do
 		owner 'root'
